crates/bevy_render/src/extract_resource.rs

```diff
@@ -48,7 +48,7 @@ pub fn extract_resource<R: ExtractResource>(
 ) {
     if let Some(mut target_resource) = target_resource {
         if main_resource.is_changed() {
-            *target_resource = R::extract_resource(&*main_resource);
+            *target_resource = R::extract_resource(&main_resource);
```

Comment:
The changes in this file look like they got pulled in by mistake :)

Reply:
(They're good changes, but we don't want to make too much of a mess of the commit history).